.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Consider specifying a more detailed version or leaving a comment explaining why version 0.2.0 is appropriate for this configuration, especially if there are newer versions available.